docs/recipes/ember.md

```diff
@@ -0,0 +1,147 @@
+# Usage with Ember
+
+Ember follows a similar pattern to [Vue](./vue.md):
+
+- The machine can be defined externally;
+- The service is placed as a property of the component;
+- State changes are observed via `service.onTransition(state => ...)`, where you set some data property to the next `state`;
+- The machine's context can be referenced as an external data store by the app. Context changes are also observed via `service.onTransition(state => ...)`, where you set another data property to the updated context;
+- The service is started (`service.start()`) when the component is created `constructor()`;
+- Events are sent to the service via `service.send(event)`.
+
+## Modern Ember (Octane)
+
+::: tip
+This example is based on Ember Octane features (Ember 3.14+)
+:::
+
+```js
+import { Machine } from 'xstate';
+
+// This machine is completely decoupled from Ember
+export const toggleMachine = Machine({
+  id: 'toggle',
+  context: {
+    /* some data */
+  },
+  initial: 'inactive',
+  states: {
+    inactive: {
+      on: { TOGGLE: 'active' }
+    },
+    active: {
+      on: { TOGGLE: 'inactive' }
+    }
+  }
+});
+```
+
+```handlebars
+<button type="button" {{on "click" (fn this.transition "TOGGLE")}}>
+  {{if this.isInactive "Off" "On"}}
+</button>
+```
+
+```js
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { interpret } from 'xstate';
+import { toggleMachine } from '../path/to/toggleMachine';
+
+export default class ToggleButton extends Component {
+  @tracked current;
+
+  get context() {
+    return this.current.context;
+  }
+
+  get isInactive() {
+    return this.current.matches('inactive');
+  }
+
+  constructor() {
+    super(...arguments);
+    this.toggleService = interpret(toggleMachine);
+    this.toggleService.onTransition(state => (this.current = state)).start();
+  }
+
+  willDestroy() {
+    super.willDestroy(...arguments);
+    this.toggleService.stop();
+  }
+
+  @action
+  transition(...args) {
+    this.toggleService.send(...args);
+  }
+}
+```
+
+## Classic Ember
```

Comment:
I wouldn't include this section.

People have no reason not to be on the latest LTS, which supports all the octane stuff.